src/components/VideoBlock/VideoBlock.tsx

```diff
@@ -65,7 +65,7 @@ const VideoBlock = (props: VideoBlockProps) => {
     const [hidePreview, setHidePreview] = useState(false);
     const norender = (!stream && !record) || !src;
     const [currentHeight, setCurrentHeight] = useState(height || undefined);
-    const fullId = `${iframeId}-${id || src}`;
+    const fullId = `${iframeId}-${id || src}${fullScreen ? '-fullscreen' : ''}`;
```

Comment:
can we use library like this - https://www.npmjs.com/package/uuid for create unique id ?

Reply:
i think yes, already, before there was a comparsion: has the video src changed or not in new compoent